planet/scripts/cli.py

```diff
@@ -219,7 +219,7 @@ async def download(ctx, order_id, quiet, overwrite, dest):
                     directory=dest,
                     overwrite=overwrite,
                     progress_bar=not quiet)
-    click.echo(f'Downloaded {len(filenames)} files.')
+    click.echo(str(filenames))
```

Comment:
@kevinlacaille strictly speaking, this is not JSON output. String representations of Python lists are in many cases isomorphic to JSON, but not always. So it would be better to echo `json.dumps(filenames)`.

Reply:
Yes, great catch - thanks!

After our discussions with @jreiberkyle we'll keep the stdout silent, but I'll keep this in mind for the future.